tests/runtime/servicers/test_global_train_servicer_impl.py

```diff
@@ -340,6 +344,8 @@ def test_global_train_Edge_Case_Widget_should_raise_when_error_surfaces_from_mod
     training_data = stream_type(
         jsondata=stream_type.JsonData(data=[SampleTrainingType(1)])
     ).to_proto()
+
+    sample_train_servicer.auto_load_trained_model = False
```

Comment:
Shoot, should have caught this one too. I think anything that mutates attributes of `sample_train_servicer` needs to be handled carefully

Reply:
ops, this came from my testing, doesn't need here. I'll remove it.